pkg/operator/ceph/csi/spec.go

```diff
@@ -392,16 +395,35 @@ func startDrivers(clientset kubernetes.Interface, rookclientset rookclient.Inter
 		}
 	}
 
-	tp.ProvisionerReplicas = 2
+	tp.ProvisionerReplicas = defaultProvisionerReplicas
 	nodes, err := clientset.CoreV1().Nodes().List(ctx, metav1.ListOptions{})
 	if err == nil {
 		if len(nodes.Items) == 1 {
 			tp.ProvisionerReplicas = 1
 		}
 	} else {
-		logger.Errorf("failed to get nodes. Defaulting the number of replicas of provisioner pods to 2. %v", err)
+		logger.Errorf("failed to get nodes. Defaulting the number of replicas of provisioner pods to %d. %v", defaultProvisionerReplicas, err)
 	}
 
+	// retrieve CSI_PROVISIONER_REPLICAS only if the node count is greater than
```

Comment:
nit: finish the comment on the same line?
```suggestion
	// retrieve CSI_PROVISIONER_REPLICAS only if the node count is greater than 1
```